content/sensu-go/5.12/reference/entities.md

```diff
@@ -151,6 +149,10 @@ spec:
 
 {{< /language-toggle >}}
 
+### Proxy entities checks
+
+Once a proxy entity is created, you can define which agents will run the check by configuring a subscription. See [Proxy Requests][11] for details on creating a proxy check for a proxy entity.
```

Comment:
As written this doesn't help the reader to connect proxy entities with proxy_requests check capability. Let's reword this to describe the connection and link to the relevant section of the check reference as well as to the "monitoring external resources" guide.

Reply:
Maybe something like this? I'm sure it's not perfect, but I thought it could be a good starting point. Everything is pulled from other docs.

Proxy entities allow Sensu to [monitor external resources][12] on systems or devices where a Sensu agent cannot be installed, like a network switch or a website. You can configure a check with a proxy entity name so that Sensu creates an entity representing the site and reports the status of the site under this entity.

After you create a proxy entity, you can define which agents will run the check by configuring a subscription. See [proxy requests][11] for details on creating a proxy check for a proxy entity.